server/supernova/server/main.cpp

```diff
@@ -182,19 +179,22 @@ void start_audio_backend(server_arguments const & args)
         output_channels = 0;
     }
 
-    cout << "opening portaudio device name: ";
-    cout << input_device << " / " << output_device << endl;
-
+    cout << "opening portaudio devices: " << endl;
+    cout << "  In: " << input_device << endl;
+    cout << "  Out: " << output_device << endl;
```

Comment:
for future reference it's good practice to only use `endl` when you mean it in C++. `endl` is equivalent to printing `'\n'` and then immediately flushing the stream. so here probably only the last one in each block of output is necessary.

Reply:
I changed first two lines to "\n", thanks for clarification.